src/main/scala/cromwell/binding/WdlExpression.scala

```diff
@@ -82,11 +91,53 @@ object WdlExpression {
     val parseTree = parser.parse_e(tokens, new WdlSyntaxErrorFormatter(terminalMap))
     new WdlExpression(parseTree.toAst)
   }
+
+  def toString(ast: AstNode, highlighter: SyntaxHighlighter = NullSyntaxHighlighter): String = {
```

Comment:
"not impossible"  I guess.  It could also just be as simple as two different HTML highlighters or something.  I guess my bigger question is... is this way of doing it really that bad?